torch/csrc/stable/stableivalue_conversions.h

```diff
@@ -342,6 +374,33 @@ struct ToImpl<torch::stable::Tensor> {
   }
 };
 
+// Specialization for StableIValue => std::vector<T>
+// std::vector<T> should be represented as a StableListHandle
+// filled with StableIValues
+// The new std::vector steals ownership of the underlying elements
+// and StableListHandle is freed.
```

Comment:
this isn't quite right -- the list the handle refers to is freed, not the handle itself, right?